### ISSUE TITLE
plugin clean folder even on re-build

### ISSUE THREAD

**Issue by shlomisas** [johnagan/clean-webpack-plugin#55]
I have this config:

```
new CleanWebpackPlugin(['dist'], {
    root: projectRoot,
    watch: false
})
```
But when the watcher run it deletes the folder and start the whole Webpack's build from scratch.
I know the `watch` flag is false by default, I added that to make sure it's settled.

I'm using Webpack version 1 due to some IE backward compatibilities, maybe this is the issue?

Thanks.

**Comment by Jonathancollinet** [comment]
look at this

### Paths (Required)
An [array] of string paths to clean
```js
[
  'dist',         // removes 'dist' folder
  'build/*.*',    // removes all files in 'build' folder
  'web/*.js'      // removes all JavaScript files in 'web' folder
]
```

**Comment by shlomisas** [comment]
i have `['dist']` as paths to clean, it's still clean it when i watch instead of just on the first web pack build.

**Comment by ianseverance** [comment]
@shlomisas As originally mentioned by @Jonathancollinet and stated in the documentation:

By passing `dist` as the `path`, you are asking for the `dist` folder itself to be cleaned (removed). To remove all of the files _within_ the `dist` folder, your config should actually be

```
new CleanWebpackPlugin([
   'dist/*.*'          // Removes all files in `dist` folder
],
{
    root: projectRoot,
    watch: false       // You can delete this, as this does not change anything for your use case
})
```
and if your `dist` folder has multiple levels, make sure to include those paths as well
```
new CleanWebpackPlugin([
   'dist/*.*'          // Removes all files in `dist` folder
   'dist/**/*.*'       // Removes all files in `dist/**` folders
   'dist/**/**/*.*'    // Removes all files in `dist/**/**` folders
],
{
    root: projectRoot,
    watch: false       // You can delete this, as this does not change anything for your use case
})
```

**Comment by chrisblossom** [comment]
Closed in [`v2.0.0`](https://github.com/johnagan/clean-webpack-plugin/tree/v2.0.0) via #99. See [johnagan/clean-webpack-plugin#usage](https://github.com/johnagan/clean-webpack-plugin#usage). Please open a new issue if you continue to have problems.
